sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinPropertyMetadata.java

```diff
@@ -28,8 +28,16 @@ public OffsetDateTime getLastUpdatedOn() {
      * @param lastUpdatedOn The date and time the property was last updated.
      * @return The DigitalTwinPropertyMetadata object itself.
      */
-    public DigitalTwinPropertyMetadata setLastUpdatedOn(OffsetDateTime lastUpdatedOn) {
+    public DigitalTwinPropertyMetadata setLastUpdatedOn(String lastUpdatedOn) {
         this.lastUpdatedOn = lastUpdatedOn;
         return this;
     }
+
+    /**
+     * Gets the date and time the property was last updated.
+     * @return The date and time the property was last updated.
+     */
+    public OffsetDateTime getLastUpdatedOnOffsetDateTime() {
```

Comment:
To compensate for not deserializing the type as a string directly, I added this method so users can still get this value as a datetime offset